### PR TITLE
feat: allow `endingTimestamp` to be 0

### PR DESCRIPTION
**Pull request by b00ste**
# What does this PR introduce?

## 🚀 Feature

Allow the validity timestamp in a relay call to start but never end

### PR Checklist

- [x] Wrote Tests
- [ ] Wrote & Generated Documentation (readme/natspec/dodoc)
- [x] Ran `npm run lint` && `npm run lint:solidity` (solhint)
- [x] Ran `npm run format` (prettier)
- [x] Ran `npm run build`
- [x] Ran `npm run test`

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-7978 Allow to not specify deadline in validityTimestamp](https://app.clickup.com/t/2645698/DEV-7978)

File: contracts/LSP25ExecuteRelayCall/LSP25MultiChannelNonce.sol [hunk 1]
```diff
@@ -114,6 +114,8 @@ abstract contract LSP25MultiChannelNonce {
             revert RelayCallBeforeStartTime();
         }
 
+        if (endingTimestamp == 0) return;
```
Comment (CJ42) [inline contracts/LSP25ExecuteRelayCall/LSP25MultiChannelNonce.sol:117]: Would be good to add some comments here to describe what is happening here.

File: tests/LSP6KeyManager/Relay/ExecuteRelayCall.test.ts [hunk 2]
```diff
@@ -1018,6 +1018,80 @@ export const shouldBehaveLikeExecuteRelayCall = (
...
+              const startingTimestamp = now - 100;
+              const endingTimestamp = 0;
+
+              const nonce = await context.keyManager.callStatic.getNonce(signer.address, 14);
+              const validityTimestamps = createValidityTimestamps(
+                startingTimestamp,
+                endingTimestamp,
+              );
+              const calldata = context.universalProfile.interface.encodeFunctionData('execute', [
+                0,
```
Comment (CJ42) [inline tests/LSP6KeyManager/Relay/ExecuteRelayCall.test.ts:1036]: let's use the `OPERATION_TYPE` from the constants here for better readability

File: tests/LSP6KeyManager/Relay/ExecuteRelayCall.test.ts [hunk 3]
```diff
@@ -1018,6 +1018,80 @@ export const shouldBehaveLikeExecuteRelayCall = (
...
+                targetContract.address,
+                0,
+                targetContract.interface.encodeFunctionData('setNumber', [nonce]),
```
Comment (CJ42) [inline tests/LSP6KeyManager/Relay/ExecuteRelayCall.test.ts:1039]: why using the nonce for the `setNumber` function? Might be better to use a separate variable, or a literal like `12345`.

Suggest to change this.

**Review by CJ42 — APPROVED** [review]

**Review by skimaharvey — APPROVED** [review]

**Pull request merged**